src/example_onchain_graph.py

```diff
@@ -0,0 +1,13 @@
+import asyncio
+from airstack.execute_query import AirstackClient
+
+
+async def main():
+    api_client = AirstackClient(api_key='ef3d1cdeafb642d3a8d6a44664ce566c')
```

Comment:
The secrets should never be hard coded. Please environment file instead for it.
If this is a key in use, you might need to delete this commit from git history to remove its traces.

Reply:
You can possibly use `python-dotenv` for this